packages/odyssey-react-mui/src/OdysseyCacheProvider.tsx

```diff
@@ -51,7 +51,9 @@ const OdysseyCacheProvider = ({
       key: uniqueAlphabeticalId,
       nonce: nonce ?? window.cspNonce,
       prepend: true,
-      speedy: false, // <-- Needs to be set to false when shadow-dom is used!! https://github.com/emotion-js/emotion/issues/2053#issuecomment-713429122
+      // TODO: Change this back!!
+      speedy: true,
+      // speedy: false, // <-- Needs to be set to false when shadow-dom is used!! https://github.com/emotion-js/emotion/issues/2053#issuecomment-713429122
```

Comment:
We should expose this on the component

Reply:
I made this implicit — if `shadowDomElement` is defined, it's `false`, otherwise it's `true`.

HOWEVER... even though we have a `shadowDomElement` prop on this component, it wasn't being used anywhere. We should investigate why.

Reply:
I found a possible answer for this; `OdysseyThemeProvider` uses (and needs) `shadowDomElement`, so I wonder if we used it in `OdysseyCacheProvider` for consistency.

Either way, it's obsolete and I'm going to deprecate it in favor of `hasShadowDomElement`.